src/utils/components/AddBootableVolumeModal/components/VolumeMetadata/components/InstanceTypeDrilldownSelect/utils/utils.ts

```diff
@@ -85,4 +87,12 @@ export const getInstanceTypeMenuItems = (
 
     return acc;
   }, initialMenuItems);
+
+  return produce(itemsData, (draftItemsData) => {
+    draftItemsData.redHatProvided.items = itemsData.redHatProvided.items.map((series) => {
+      return produce(series, (seriesDraft) => {
```

Comment:
why those produce? what benefit do we get here from using produce inside a simple function? immer , etc , is usually good for shared objects that you don't want to mutate, such as the VM object. here you create itemsData... as reduce return new object.. i don't see any benefit for it